terraform/LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
Do we really need the duplicate `LICENSE` file?

Reply:
I was removing the LICENSE file but missed it in here. Good catch, thanks.

Reply:
Done